sdk/eventhub/azure-eventhub/azure/eventhub/_connection_manager.py

```diff
@@ -30,9 +30,8 @@ class ConnectionManager(Protocol):
         def get_connection(
             self,
             *,
-            host: Optional[str] = None,
+            endpoint: str,
             auth: Optional[Union[JWTTokenAuth, uamqp_JWTTokenAuth]] = None,
-            endpoint: Optional[str] = None,
```

Comment:
just to confirm host and endpoint were the same here?

Reply:
yeah I did a bit of consolidation here, [uamqp](https://github.com/Azure/azure-uamqp-python/blob/74e469c1572be6366eeb52e519e9cf749b7d7816/uamqp/connection.py#L62) calls it hostname while [azure amqp](https://github.com/Azure/azure-sdk-for-python/blob/d2e868ceda25d4483e306ea166693af80ef229a0/sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/_connection.py#L95) calls it endpoint. 

So I made the change in the transport ABC to make them one and the same and get rid of some pylint disables.